modules/nf-core/picard/addorreplacereadgroups/main.nf

```diff
@@ -24,26 +24,24 @@ process PICARD_ADDORREPLACEREADGROUPS {
     script:
     def args = task.ext.args        ?: ''
     def prefix = task.ext.prefix    ?: "${meta.id}"
-    def suffix = task.ext.suffix    ?: "${bam.getExtension()}"
+    def suffix = task.ext.suffix    ?: "${reads.getExtension()}"
     def reference = fasta ? "--REFERENCE_SEQUENCE ${fasta}" : ""
-    def create_index = ( suffix == "bam" )? "--CREATE_INDEX" : ""
```

Comment:
so no creating index anymore?

Reply:
Yes to creating index, but it's a flag that should be set by the user in `ext.args`

Reply:
I added `--CREATE_INDEX` to `ext.args` in one of the test configs just so we keep testing index generation works if configured and that the bai is captured in the proper output